### DISCUSSION TITLE
Spacy/ thic.api

### DISCUSSION THREAD

**Discussion by kristinepetrosyan** [explosion/spaCy#7695] [Help: Installation] [answered]
I installed Prodigy and SpaCy. Getting error <ImportError: cannot import name 'get_array_module' from 'thinc.api'>
Tried all versions but no luck

## Your Environment
Info about spaCy

* **spaCy version:** 2.3.2
* **Platform:** Darwin-19.0.0-x86_64-i386-64bit
* **Python version:** 3.7.6

**Comment by adrianeboyd** [comment, marked as answer]
It sounds like you may have ended up with a mixture of spacy v2 and spacy v3 dependencies, possibly from a downgrade that didn't run cleanly.

Our best suggestion is to try again in a brand new venv, installing prodigy directly without upgrading or downgrading spacy, which doesn't always work correctly with `pip`. Try something like this:

```bash
python -m venv .venv
source .venv/bin/activate
pip install -U pip setuptools wheel
pip install prodigy-*.whl
```

prodigy should automatically pull in the correct versions of spacy and all its dependencies.

**Reply by kristinepetrosyan** [reply]
Thank you for helping  @adrianeboyd . It worked!
